modules/message.sh

```diff
@@ -231,6 +231,34 @@ function message {
             log_comment_and_exit1 "Error: You do not have an authorized access token" "To get access token, run $BASENAME message gitter apitoken"
           fi
           ;;
+        "")
+          if check_apitoken gitter; then
+            get_apitoken gitter
+            echo
+            echo "You have successfully configured your Gitter apitoken."
+            echo "Now you must provide a subcommand for $BASENAME message gitter listed below:"
```

Comment:
This could also summarize the subcommands in one line:

e.g.
> "Now you must provide a subcommand for $BASENAME message gitter [channels|send|show|read|mark] listed below:"